internal_ws/ykcompile/src/lib.rs

```diff
@@ -1386,7 +1386,9 @@ impl TraceCompiler {
         //
         // FIXME Lukas, what's the `16 - ` for here again?
         let topalign = 16
-            - (live_off + soff as usize + CALLEE_SAVED_REGS.len() * REG_SIZE)
+            - (live_off
+                + soff as usize
+                + CALLEE_SAVED_REGS.len() * usize::try_from(QWORD_REG_SIZE).unwrap())
```

Comment:
This is a `usize` so the conversion seems pointless?

Reply:
Oops.

It's because I toyed with making our constant an `i32` so that it could be used with dynasm directly, but that caused other places to require casting.

I hate juggling integer types in rust. I wish `as` was actually a "try_from().unwrap()".

Reply:
> I hate juggling integer types in rust. I wish as was actually a "try_from().unwrap()".

I wish `as` didn't exist :p